### DISCUSSION TITLE
for DEMOLITIONDON96

### DISCUSSION THREAD

**Discussion by ErenKaya31** [DEMOLITIONDON96/Demolition-Engine#36] [General (Friday Night Chattin')]
1. i am not rude
2. fix your things that doesnt fixxed
3. i love you (like friends im not gay)

**Comment by ThatOneSillyGuyDon** [comment]
we already fixed the marvelous rating, stages in lua are the exact same as in Psych, for it's still mostly dependent on Psych atm, Damage Notes are fixed in Pixel Stages, basically, just get 0.1.1, for it has all the fixes done for it

**Reply by RealDealJinx** [reply]
now love him (like friends ErenKaya31 is not gay)
